solution/zenko-version-cr.yaml

```
@@ -0,0 +1,43 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: 1.1-dev
+spec:
+  imagePullPolicy: Always
+  versions:
+    cloudserver:
+      image: cloudserver
+      tag: latest-8.1.2
+    backbeat:
+      image: backbeat
+      tag: 161b9c1
+    blobserver:
+      image: blobserver
+      tag: 1.0.8
+    jabba:
+      image: jabba
+      tag: 940d
+    secureChannelProxy:
+      image: cloudserver
+      tag: 4332d0a
+    localData:
+      image: cloudserver
+      tag: 8.1.15
+    metrics:
+      image: cloudserver
+      tag: 4332d0a
+    s3utils:
+      image: s3utils
+      tag: 1.2
+    zookeeper:
+      image: zookeeper
+      tag: 0.2.4
+    kafka:
+      image: kafka
+      tag: 2.12-2.3.0
+    shell:
+      image: busybox
+      tag: 1.31.0-glibc
+    mongodb: 4.0
```

hadn't notice this earlier (needs to match the image tag in the dep list)
```suggestion
    mongodb: 4.0-v1
```

---

oh yea oops, I had changed it to just 4.0 as it wasnt pulling before. Will accept this. ( and make sure I change the commit message properly this time... )